ReportSimulationOutput/measure.rb

```diff
@@ -137,6 +137,7 @@ def energyPlusOutputRequests(runner, user_arguments)
     super(runner, user_arguments)
 
     result = OpenStudio::IdfObjectVector.new
+    return result if runner.halted
```

Comment:
This doesn't seem like the right place for this code. It shouldn't be a measure's responsibility to check for runner.halted; have you ever seen any other measure with this check here?

If I had to guess, I assume the check should be inside meta_measure.rb? Maybe in [here](https://github.com/NREL/OpenStudio-HPXML/blob/27ca8d6323aef936ae312d64179cc61fe8c70128/HPXMLtoOpenStudio/resources/meta_measure.rb#L189) and [here](https://github.com/NREL/OpenStudio-HPXML/blob/27ca8d6323aef936ae312d64179cc61fe8c70128/HPXMLtoOpenStudio/resources/meta_measure.rb#L213)?

Reply:
SimulationOutputReport on resstock develop has this check: https://github.com/NREL/resstock/blob/develop/measures/SimulationOutputReport/measure.rb#L54.

For bsb where the workflow is generated up front, I'm not sure how we would remove SimulationOutputReport from the workflow after the workflow has already begun? In reality, this may be an issue in the OS workflow gem (`run` methods get skipped but `energyPlusOutputRequests` methods do not.

Reply:
Okay, I see the issue you're facing. I guess there's no harm in adding this code to the measure, but yes, it seems like a bug should be filed against the OS workflow gem.

Reply:
I created an issue on OpenStudio-workflow-gem: https://github.com/NREL/OpenStudio-workflow-gem/issues/143